velox/docs/functions/spark/datetime.rst

```diff
@@ -312,8 +317,13 @@ These functions support TIMESTAMP and DATE input types.
     `Datetime patterns for formatting and parsing
     <https://spark.apache.org/docs/latest/sql-ref-datetime-pattern.html>`_.
     Returns null if ``string`` does not match ``format`` or if ``format``
-    is invalid. When `Simple` date formatter is used, permit partial date
-    parsing; otherwise, exception is thrown.
+    is invalid. The `Simple` date formatter permits partial date parsing
```

Comment:
Thank you for clarifying the docs. I see that these docs repeat which creates a risk of them diverging. Would it be possible to remove copy-paste and refer to 'get_timestamp' function documentation. Alternatively, we may create a separate section that explains the 'format' and refer to it from all functions that use that 'format'.

Reply:
@mbasmanova There are some different behaviors between `Simple` and `Joda` date formmater including how they handle invalid formats, whether the entire input needs to be consumed, and so on.  How about creating a new section to specifically clarify these differences?

Reply:
> How about creating a new section to specifically clarify these differences?

That sounds good. Thanks.

Reply:
@mbasmanova I have revised the documentation about the `Simple` date formatter for clarity and conciseness.  Could you please take a look?